app/client/rest/preferences.ts

```diff
@@ -9,6 +9,7 @@ export interface ClientPreferencesMix {
 
 const ClientPreferences = (superclass: any) => class extends superclass {
     savePreferences = async (userId: string, preferences: PreferenceType[]) => {
+        this.analytics.trackAPI('api_preferences_save_post');
```

Comment:
For line 12 and 27, I need confirmation confirm if we have to track this analytics.  The event on v1 was `action_posts_flag` but I renamed it here.  Please let me know your thoughts.

Reply:
keep the same name as it was..

Reply:
done